aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -76,9 +76,11 @@ Tensor _to_copy(
     }
   }
   // See Note [Explicit nullopt MemoryFormat argument]
-  auto r = at::empty(self.sizes(),
-                     options.memory_format(memory_format).pinned_memory(pin_out),
-                     c10::nullopt);
+  // TODO: empty_quantized does not work here. It raises an exception in CheckMemoryFormat.h prior to
+  // empty_affine_quantizd/_empty_per_channel_affine_quantized calls
+  auto r = self.is_quantized() ? at::empty_like(self, memory_format)
```

Comment:
can we explain the reason that we are using `empty_like` here as well? can just copy from summary